app/src/main/res/layout/cl_fragment_signup.xml

```diff
@@ -0,0 +1,329 @@
+                        style="@style/TextViewCommonStyle"
+                        android:text="@string/phone"
+                        app:layout_constraintBottom_toTopOf="@id/et_phone"
+                        app:layout_constraintStart_toStartOf="@id/cv_phone_layout"
+                        app:layout_constraintTop_toTopOf="@id/cv_phone_layout"/>
+
+                    <com.example.internproject_revamp.widgets.CLEditText
+                        android:id="@+id/et_phone"
+                        style="@style/EditTextCommonStyle.ProfileEditText"
+                        android:hint="@string/mobile_number"
+                        android:maxLength="10"
+                        app:layout_constraintBottom_toBottomOf="@id/cv_phone_layout"
+                        app:layout_constraintEnd_toEndOf="@id/cv_phone_layout"
+                        app:layout_constraintStart_toStartOf="@id/cv_phone_layout"
+                        app:layout_constraintTop_toBottomOf="@id/tv_phone"/>
+
+                </androidx.constraintlayout.widget.ConstraintLayout>
+            </androidx.cardview.widget.CardView>
+
+            <androidx.cardview.widget.CardView
+                android:id="@+id/cv_password"
+                style="@style/SignupCardViewStyle"
+                app:layout_constraintEnd_toEndOf="@+id/cl_layout"
+                app:layout_constraintStart_toStartOf="@+id/cl_layout"
+                app:layout_constraintTop_toBottomOf="@id/cv_phone">
+
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:id="@+id/cv_password_layout"
+                    style="@style/EditProfileConstraintLayoutStyle">
+
+                    <TextView
+                        android:id="@+id/tv_password"
+                        style="@style/TextViewCommonStyle"
+                        android:text="@string/password"
+                        app:layout_constraintBottom_toTopOf="@id/et_password"
+                        app:layout_constraintStart_toStartOf="@id/cv_password_layout"
+                        app:layout_constraintTop_toTopOf="@id/cv_password_layout"/>
+
+                    <com.example.internproject_revamp.widgets.CLEditText
+                        android:id="@+id/et_password"
+                        style="@style/EditTextCommonStyle.ProfileEditText"
+                        android:hint="@string/password"
+                        android:inputType="textVisiblePassword"
+                        app:layout_constraintBottom_toBottomOf="@id/cv_password_layout"
+                        app:layout_constraintEnd_toEndOf="@id/cv_password_layout"
+                        app:layout_constraintStart_toStartOf="@id/cv_password_layout"
+                        app:layout_constraintTop_toBottomOf="@id/tv_password"/>
+
+                </androidx.constraintlayout.widget.ConstraintLayout>
+            </androidx.cardview.widget.CardView>
+
+            <androidx.cardview.widget.CardView
```

Comment:
Seems we have a similar kind of UI here. check with me regarding them to move as custom layouts